velox/connectors/hive/storage_adapters/gcs/tests/GCSFileSystemTest.cpp

```diff
@@ -16,170 +16,30 @@
+namespace {
 class GCSFileSystemTest : public testing::Test {
- protected:
-  static void SetUpTestSuite() {
-    if (testbench_ == nullptr) {
-      testbench_ = std::make_shared<GcsTestbench>();
-    }
-
-    ASSERT_THAT(testbench_, ::testing::NotNull());
-    ASSERT_THAT(testbench_->error(), ::testing::IsEmpty());
-
-    // Create a bucket and a small file in the testbench. This makes it easier
-    // to bootstrap GcsFileSystem and its tests.
-    auto client = gcs::Client(
-        google::cloud::Options{}
-            .set<gcs::RestEndpointOption>(
-                "http://localhost:" + testbench_->port())
-            .set<gc::UnifiedCredentialsOption>(gc::MakeInsecureCredentials()));
-
-    bucket_name_ = "test1-gcs";
-    google::cloud::StatusOr<gcs::BucketMetadata> bucket =
-        client.CreateBucketForProject(
-            bucket_name_, "ignored-by-testbench", gcs::BucketMetadata{});
-    ASSERT_TRUE(bucket.ok()) << "Failed to create bucket <" << bucket_name_
-                             << ">, status=" << bucket.status();
-
-    object_name_ = "test-object-name";
-    google::cloud::StatusOr<gcs::ObjectMetadata> object =
-        client.InsertObject(bucket_name_, object_name_, kLoremIpsum);
-    ASSERT_TRUE(object.ok()) << "Failed to create object <" << object_name_
-                             << ">, status=" << object.status();
-  }
-
-  std::shared_ptr<const config::ConfigBase> testGcsOptions() const {
-    std::unordered_map<std::string, std::string> configOverride = {};
-
-    configOverride["hive.gcs.scheme"] = "http";
-    configOverride["hive.gcs.endpoint"] = "localhost:" + testbench_->port();
-    return std::make_shared<const config::ConfigBase>(
-        std::move(configOverride));
-  }
-
-  std::string preexistingBucketName() {
-    return bucket_name_;
-  }
-
-  std::string preexistingBucketPath() {
-    return bucket_name_ + '/';
-  }
-
-  std::string preexistingObjectName() {
-    return object_name_;
-  }
-
-  std::string preexistingObjectPath() {
-    return preexistingBucketPath() + preexistingObjectName();
+ public:
+  void SetUp() {
+    testbench_ = std::make_shared<GCSTestbench>();
+    testbench_->bootstrap();
   }
-  static void TearDownTestSuite() {}
 
-  static std::shared_ptr<GcsTestbench> testbench_;
-  static std::string bucket_name_;
-  static std::string object_name_;
+  std::shared_ptr<GCSTestbench> testbench_;
 };
 
-std::shared_ptr<GcsTestbench> GCSFileSystemTest::testbench_ =
-    nullptr; // will be destroyed on destructor
-std::string GCSFileSystemTest::bucket_name_;
-std::string GCSFileSystemTest::object_name_;
-
 TEST_F(GCSFileSystemTest, readFile) {
-  const std::string gcsFile =
-      gcsURI(preexistingBucketName(), preexistingObjectName());
+  const auto gcsFile = gcsURI(
+      testbench_->preexistingBucketName(), testbench_->preexistingObjectName());
 
-  filesystems::GCSFileSystem gcfs(testGcsOptions());
+  filesystems::GCSFileSystem gcfs(testbench_->hiveConfig());
```

Comment:
same here. Acronyms are not all upper cases (`GcsFileSystem`)

Reply:
fixed everywhere.